docs/search_and_race/search_algorithm_switching.md

```diff
@@ -1,37 +1,531 @@
 # Search Algorithm Switching
 
-As part of the Search and Race project work, we've decided to explore allowing users to
-submit their searches using different search algorithms.
+As part of the Search and Race project work, we've decided to explore allowing users to execute their catalog search queries using alternative algorithms used for querying Apache Solr as well as for ranking the search results.
 
-## Context
+## Technical Implementation
 
-We could do this primarily in ruby code or primarily in Solr.
+### Apache Solr Request Handler
 
-### Solr Request Handler
+_(Please refer to [Request handler search component documentation'](https://solr.apache.org/guide/solr/latest/configuration-guide/requesthandlers-searchcomponents.html#RequestHandlersandSearchComponentsinSolrConfig-RequestHandlers) for this component of Apache Solr.)_
 
-Define weights in the solrconfig. Use a sew SearchBuilder to submit searches with "qt=different_algorithm" instead of "qt=search".
-* Pros:
-    * Maximum functionality
-    * hides a bunch of parameters.
-* Cons:
-    * Have to maintain two request handlers, have to deploy.
+In order to provide support at the level of the SolrCloud deployment for the
+Princeton University Library, one must please first focus upon proposing changes
+for the [PUL Solr](https://github.com/pulibrary/pul_solr) GitHub repository.
 
-### Blacklight Request Parameters
+An example of this may be found within the
+[Solr cloud collection, solrconfig.xml file'](https://github.com/pulibrary/pul_solr/blob/main/solr_configs/catalog-production-v2/conf/solrconfig.xml). In this case, the `requestHandler` `engineering_search` was added, however, for the purposes of this document, the
```

Comment:
Should this be changed to point to a specific commit, since we're planning on removing this from main?